### PR TITLE
2024 Craig User Test Edits

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: web-ui.markdown [hunk 1]
```diff
@@ -92,16 +92,15 @@ All Events can be searched and viewed from the Event Log page.
...
 
 
-### Host count widget
+### Newly bootstrapped hosts widget
 
-The hosts count widget helps to visualize the number of hosts bootstrapped to CFEngine over time.
+The Newly bootstrapped hosts widget helps to visualize the number of hosts bootstrapped to CFEngine over time.
 
-<img src="dashboard-widget-hosts-count.png" alt="Dashboard Host count" width="412px">
+TODO: update screenshot <img src="dashboard-widget-hosts-count.png" alt="Dashboard Host count" width="412px">
```
Comment (olehermanse) [inline web-ui.markdown:99]:
```suggestion
<img src="dashboard-widget-hosts-count.png" alt="Dashboard Host count" width="412px">
```

File: web-ui.markdown [hunk 2]
```diff
@@ -27,18 +27,18 @@ The dashboard contains informative widgets that you can customize to create
 alerts. All notifications of alert state changes, e.g. from OK to not-OK, are
 stored in an event log for later inspection and analysis.
 
-### Make changes to shared dashboard
+### Make changes to a shared dashboard
 
-<img src="clone-dashboard.png" alt="Clone dashboard possibility" width="490px">
+TODO: replace <img src="clone-dashboard.png" alt="Clone dashboard possibility" width="490px">
```
Comment (olehermanse) [inline web-ui.markdown:32]:
```suggestion
<img src="clone-dashboard.png" alt="Clone dashboard possibility" width="490px">
```

**Review by olehermanse — APPROVED** [review]

**Review by nickanderson — APPROVED** [review]

**Comment by nickanderson** [comment]
@cf-bottom jenkins please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/599/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/599/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-599/output/_site/

**Comment by nickanderson** [comment]
I see these changes in 3.24, and 3.21 branches. No cherry picks needed.

**Pull request merged**